docs/src/intro.rst

```diff
@@ -15,7 +15,7 @@ of computational materials science and chemistry.
 
 
 
-- Fingerprint Selection:
+- Features and Samples Selection:
```

Comment:
Was this an intentional change?

Reply:
Yes - the text and methods below refers to both features and samples, so it seemed more consistent to mention also samples in the title.